plugins/reveal.js

```diff
@@ -82,7 +82,8 @@ class Reveal {
   }
 
   nextSlide() {
-    return this.page.evaluate(_ => Reveal.next());
+    this.page.evaluate(_ => Reveal.next());
+    return this.page.evaluate(_ => Reveal.layout());
```

Comment:
I wonder if this is really needed to call it for every slide. Could you please check if calling it once in the `configure` method solve your issue?

Reply:
Will check, thx for the quick feedback.